lua/ulx/modules/sh/pcrestrict.lua

```diff
@@ -0,0 +1,45 @@
+CATEGORY_NAME = "CFC"
```

Comment:
Are we sure that all of this needs to be in a shared file?

Reply:
Yeh